readme.md

```diff
@@ -1,34 +1,57 @@
 
-## Features
+## Welcome!
 
-### Discover, register and manage your Industrial Assets with Azure
+The Azure Industrial IoT Cloud Platform is a Microsoft product that has fully embraced openness. We use Azure's managed Platform as a Service (PaaS) services, open-source software leveraging the MIT license throughout, open international standards for communication (OPC UA, MQTT) and interfaces (Open API) and open industrial data models (OPC UA) on the edge and in the cloud. This means you are **not** locked in to our platform when using it!
 
-Azure Industrial IoT allows plant operators to discover [OPC UA](docs/opcua.md) enabled assets in a factory network and register them in Azure IoT Hub.  
+## Discover, register and manage your industrial assets with Azure
 
-### Analyze, react to events, and control factory equipment from anywhere
+The Azure Industrial IoT Cloud Platform allows you to discover industrial assets on-site and automatically registers them in the cloud for easy access there. It leverages managed Azure PaaS services so you don't have to manage the platform yourself. On top of the Azure PaaS services, we have built a number of edge and cloud services that work in harmony, leveraging OPC UA as the data model. This is also the first cloud platform to leverage the OPC UA PubSub telemetry format (both JSON and binary, on top of MQTT). If your assets don't support OPC UA as an interface, we have worked with our large partner network to support all types of industrial interfaces through the use of adapters, fully integrated with our platform. Please check out the [Azure IoT Edge Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/category/internet-of-things?page=1&subcategories=iot-edge-modules) for a list of adapters available.
 
-Operations personnel can subscribe to and react to events on the factory floor from anywhere in the world.  The Microservices' REST APIs provide most of the defined OPC UA services and are secured using OAUTH authentication and authorization backed by Azure Active Directory (AAD).  This enables your cloud applications to browse server address spaces or read/write variables and execute methods using HTTPS as well as process OPC UA Pub/Sub messages.  
+The edge services are implemented as Azure IoT Edge modules and run on on-premises [Kubernetes](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-install-iot-edge-kubernetes), or stand-alone on Docker. The  cloud services are implemented as ASP.NET micro-services with a REST interface and run on managed Azure Kubernetes Services (AKS) or stand-alone on Azure App Service . For both edge and cloud services, we have provided pre-built Docker containers in the Microsoft Container Registry (MCR), so you don't have to build them yourself. The edge and cloud services are leveraging each other and must be used together. We have also provided easy-to-use deployment scripts that allow you to deploy the entire platform in a step-by-step fashion.
```

Comment:
Kubernetes is not yet supported by our components.   We should make sure this is first tested and we have documentation for it.

Reply:
Space after Azure App Service .